src/main/java/emily/command/Emily.java

```diff
@@ -0,0 +1,68 @@
+package main.java.emily.command;
+
+import java.io.File;
+import java.util.Scanner;
+
+/**
+ * Main program
+ */
+public class Emily {
+
+    private static final String FILE_PATH = "data/emily.txt";
+    private final Storage storage;
+    private TaskList tasks;
+    private final Ui ui;
+
+    /**
+     * Initialises Emily bot
+     * @param filePath location to store the information
+     */
+    public Emily(String filePath) {
+        ui = new Ui();
+        storage = new Storage(filePath);
+        try {
+            tasks = new TaskList(storage.readData());
+        } catch (DukeException e) {
+            File f = new File(FILE_PATH);
+            f.getParentFile().mkdirs();
+        }
+    }
+
+    /**
+     * Interact with the user
```

Comment:
I think we should use the word "Interacts" instead of "Interact"? I noticed the same issue in a few other places as well.